WebPhapp/client/package.json

```diff
@@ -3,10 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.5.1",
+    "axios": "^0.18.0",
+    "prop-types": "^15.6.2",
     "react": "^16.6.0",
+    "react-data-grid": "^4.0.9",
     "react-dom": "^16.6.0",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.1"
+    "react-scripts": "2.1.1",
+    "react-table": "^6.8.6",
+    "typeface-titillium-web": "0.0.54"
```

Comment:
Unused import. Can be removed.

Reply:
Can you clarify which import(s)? Typeface-titillium-web is in use for the font of each prescription row? However, I did remove react-table.

Reply:
Typeface-titillium-web. Try running it with and without the import and compare the fonts. It looks the same to me. The index.css seems to link it externally, but I could be wrong. If so, I'll drop it :relaxed:

Reply:
You are right. I removed the package. The font is changed by line 1 in /client/src/index.css